### DISCUSSION TITLE
Rebuilding your cache, please be patient -- hanging

### DISCUSSION THREAD

**Discussion by Jacka12** [kimai/kimai#3476] [Questions & Answers]
I have problem when build the Kimai2 on my server 
   - I user helm deploy in k8s. Problem is when i use with PV hostPath, it will running Ok. But when i use Pv with NFS . It hang in at  Rebuilding your cache, please be patient.

**Comment by ghost** [comment]
Hello,

maybe your hosting environment is running out of memory so the task fails.
